components/server/src/grpc/users.rs

```diff
@@ -68,15 +69,29 @@ impl UserService for UserServiceImpl {
             self.authorizer.check_unregistered_oidc(&token).await,
             "Unauthorized"
         );
-        let user_id = tonic_internal!(
+        let user = tonic_internal!(
             self.database_handler
                 .register_user(request, external_id)
                 .await,
             "Internal register user error"
         );
 
+        // Try to send mail to recently registered users email address
+        if let Some(mailclient) = self.mailclient.as_ref() {
+            let _ = mailclient.send_message(
+                &user.email,
+                format!("Dear {},
+We are excited to inform you that your registration with Aruna has been successfully completed.\n
+After your account has been activated by an administrator you can start exploring all the features we offer. Should you need any assistance or have any questions, you can create support tickets by reaching out to us at support@aruna-storage.org.\n
```

Comment:
Add missing comma: 
"After your account has been activated by an administrator, you can start exploring all the features we offer. [...]"

Reply:
Next time you can simply make a suggestion that I only need to accept. 
Simplifies the process :v: